components/DesaDigital/Detail/index.vue

```diff
@@ -0,0 +1,240 @@
+<template>
+  <div class="desa-digital-detail">
+    <div class="desa-digital-detail__action">
+      <BaseButton variant="primary" @click="goBack">
+        <template #icon>
+          <div class="desa-digital-detail__action-button">
+            <div class="desa-digital-detail__action-icon">
+              <IconArrowLeft />
+            </div>
+            Kembali
+          </div>
+        </template>
+      </BaseButton>
+    </div>
+    <div class="desa-digital-detail__content">
+      <h1 class="desa-digital-detail__content-header">
+        Identitas Pemohon
+      </h1>
+      <jds-simple-table class="desa-digital-detail__content-table">
+        <tbody>
+          <tr>
+            <td class="desa-digital-detail__content--width">
+              <strong>Nama Lengkap</strong>
+              <span>:</span>
+            </td>
+            <td>{{ item.properties.applicant.name || '-' }}</td>
```

Comment:
Hello, @doohanas please be careful accessing a deeply nested object without proper error handling, the caveat of using an Object as a prop is we as a developer often doesn't have control of what the data structure looks like (though most likely it will represent the API contract). 

One way I can think of is to create a computed property for each of the properties we want to access and add optional chaining (?) to make sure we don't throw an error 

```js
const item = {
  properties: null
}

console.log(item.properties.applicant.name || '-') // TypeError: Cannot read properties of null (reading 'applicant')
console.log(item.properties?.applicant?.name || '-') // output: -
```

Reply:
as i know Optional chaining can only used inside script tag, can't be used inside html (template) tag.. #cmiiw @Ibwedagama

Reply:
Yes you're right @yoslie that's why I suggest using a computed property to render them 👍🏼

Reply:
So, we move for accessing the object data in computed and use optional chaining?

Reply:
I think that's the safest way to do it, let's say the API returns an error (Server down or something went wrong), and the frontend app would throw an error because the object is null.

But this is just a suggestion, maybe there is a better way to do it 🤔 

what do you think? @doohanas @yoslie

Reply:
I agree with you, If the server goes down or has trouble will throw an error.

So, what if I write like this? I combine it with `Nullish coalescing operator` to add a value if the object data is undefined or null. Wdyt?

```js
    bumdes () {
      return this.item.properties?.bumdes?.bumdes.data ?? '-'
    }
```

Reply:
Yeah, I think that's maybe the safest way to handle the error if something happens to the API, it does make our codebase a little bit bigger tho 😅 @doohanas

Reply:
The change [here](https://github.com/jabardigitalservice/dedi-cms/pull/92/commits/cf30c10565f755dc2a93a64d7a518e9a76a73a28) ⭐